### PR TITLE
Add go.mod, changelog, and travis-ci windows worker

### PR DESCRIPTION
**Pull request by andrewkroh**
- Use go.mod the list the project deps.
- Add a changelog.
- Update readme with more info.
- Test project on Travis-CI Windows worker.
- Ignore "no such process" errors in integration test.

### CODE REVIEW

File: .ci/scripts/check_lint.go [hunk 1]
```diff
@@ -30,57 +31,54 @@ import (
...
 	if len(out) > 0 {
+		log.Println("There are golint warnings.")
 		fmt.Printf(string(out))
 		os.Exit(1)
 	}
 }
 
 func filterIgnores(out []byte) ([]byte, error) {
-	var buffer bytes.Buffer
+	var lines [][]byte
```
Comment (cwurm) [inline .ci/scripts/check_lint.go]: Personally, I find this function harder to read after the change. More lines of code, and more complex with the use of a nested array.

**Comment by cwurm** [comment]
Nice to see how modules reduce the size of this repo.

Are there any changes necessary in projects that use go-sysinfo as a dependency, e.g. Beats?

File: .ci/scripts/check_lint.go (hunk 1 above)
Comment (andrewkroh) [reply .ci/scripts/check_lint.go]: I copied it from another repo. But I agree the old version seemed simpler. I'll revert this. Hopefully there's no functional difference. I wish there was no need for any of this.

**Comment by andrewkroh** [comment]
> Are there any changes necessary in projects that use go-sysinfo as a dependency, e.g. Beats?

This shouldn't affect Beats. But after we make a release then Beats can depend on tagged releases. Projects that use `go mod` (or tools that understand go.mod) will have the benefit being able to follow the transitive dependencies.

**Pull request merged**